.claude/skills/hook-creator/references/guide.md

```
@@ -0,0 +1,500 @@
+# 차단 (stderr에 이유 출력)
+echo '{"decision": "deny", "reason": "위험한 경로 감지"}' >&2
+exit 2
+
+# 사용자에게 메시지 전달
+echo '{"continue": true, "systemMessage": "경고: 보안 파일 수정됨"}' >&2
+exit 0
+```
+
+### prompt 훅 응답
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)